typedb/stream/response_collector.py

```diff
@@ -46,56 +46,71 @@ def new_queue(self, request_id: UUID):
     def get(self, request_id: UUID) -> Optional["ResponseCollector.Queue[R]"]:
         return self._collectors.get(request_id)
 
-    def close(self, error: Optional[RpcError]):
+    def remove(self, request_id: UUID):
+        with self._collectors_lock:
+            del self._collectors[request_id]
```

Comment:
Something feels awkward about this - not the fault of the PR but rather the existing code.

We have a class named `ResponseCollector`, which contains an object named `_collectors`. This is simply illogical since the type of those collectors is not `ResponseCollector`.

Do you have any idea how we might improve the terminology?

Reply:
It's a fair point, I think the idea is to see the `ResponseCollector` as a single thing - which means the naming inside is off - renamed to `_response_queues`